failOnVersionConflict.gradle

```diff
@@ -0,0 +1,137 @@
+/**
+ * This task will parse all dependency files in 'build/deps' and tries to find version conflicts.
+ * Our assumption is: Version should be aligned by enforcing versions from our platform module
+ * 'sda-commons-dependencies'. We don't care if we find different versions as long as Gradle is
+ * able to choose the right version from 'sda-commons-dependencies'.
+ */
+task failOnVersionConflict {
```

Comment:
Should this be added somewhere in the GH-Actions?

Reply:
The task will be executed by the following lines of code at the end of the file:

```
// Make sure we can find the dependency files for all subprojects
failOnVersionConflict.dependsOn(subprojects.allDeps)

// execute task with 'check'
check.dependsOn failOnVersionConflict 
```